platform/docs/docs/configuration/theme-configuration.md

```diff
@@ -0,0 +1,287 @@
+# Theme Configuration
```

Comment:
Is `theme` a config-point concept? since `theme` in OHIF might mean appearance ... 
CC: @swederik

Reply:
The theme settings are mostly around appearance, so I called it theme.  If you want another name, thats fine with me, just let me know what it is.  I think of them as themes, even if it is somewhat more than just appearance, it still tends to be related to how things look and interact to the user.